sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -349,25 +359,39 @@ def send_messages(self, message, **kwargs):
         timeout = kwargs.pop("timeout", None)
         if timeout is not None and timeout <= 0:
             raise ValueError("The timeout must be greater than 0.")
-        message = transform_messages_to_sendable_if_needed(message)
-        try:
-            batch = self.create_batch()
-            batch._from_list(message)  # pylint: disable=protected-access
-            message = batch
-        except TypeError:  # Message was not a list or generator.
-            pass
-        if isinstance(message, BatchMessage) and len(message) == 0:  # pylint: disable=len-as-condition
-            raise ValueError("A BatchMessage or list of Message must have at least one Message")
-        if not isinstance(message, BatchMessage) and not isinstance(message, Message):
-            raise TypeError("Can only send azure.servicebus.<BatchMessage,Message> or lists of Messages.")
-
-        self._do_retryable_operation(
-            self._send,
-            message=message,
-            timeout=timeout,
-            operation_requires_timeout=True,
-            require_last_exception=True
-        )
+
+        with send_trace_context_manager() as send_span:
```

Comment:
for myself learning: we didn't set `span_name` in `cancel_schedule`, is it by design?
(in `constants.py`, I didn't see something like `SPAN_NAME_CANCEL_SCHEDULE = TRACE_NAMESPACE + '.cancel_schedule'`)